src/routes/user/register/1.svelte

```diff
@@ -0,0 +1,375 @@
+<script lang="ts">
+	import type { page_prop } from './types.ts';
+
+	import { createEventDispatcher } from 'svelte';
+	import Markdown from '$components/markdown.svelte';
+	import * as _ from 'lodash-es';
+	import { z } from 'zod';
+	import { handle_input } from '$functions/forms/handle_input';
+	import { zxcvbn, zxcvbnOptions, type OptionsType } from '@zxcvbn-ts/core';
+	import { cn } from '$functions/classnames';
+	import { FETCH_TIMEOUT } from '$constants/fetch';
+	import { autofocus } from '$functions/forms/autofocus';
+	import Arrow from '$icons/shapes/arrow.svelte';
+	import Info from '$icons/shapes/info.svelte';
+	import Tick from '$icons/shapes/tick.svelte';
+	import ArrowUpRight from '$icons/shapes/arrow_up_right.svelte';
+	import CoreText from '$icons/text/core.svelte';
+
+	let pages_state = new Array({});
+
+	let combined_state: { [key: string]: string };
+	$effect.pre(() => {
+		console.log(pages_state);
+		combined_state = Object.assign({}, ...pages_state);
+	});
```

Comment:
Its because `bind:value={some_state}` this **some_state** needs to be defined (dont be undefined).
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/35/commits/f35fb0641d4596e4ac6f93edc8a18d6bab3c9f2a